src/common/rpc/RequestBuilder.jl

```diff
@@ -396,28 +392,36 @@ function create_req(label::Label)
     )
 end
 
-function get_relates_req(label::Label, role_label::Optional{String})
+function get_relates_req(label::Label)
     return _treq(label.name, label.scope;
-        relation_type_get_relates_req = Proto.RelationType_GetRelates_Req(;
-            label = role_label
-        )
+        relation_type_get_relates_req = Proto.RelationType_GetRelates_Req()
     )
 end
 
 function set_relates_req(
-    label::Label, role_label::String, overridden_label::Optional{String}
-)
+    label::Label, role_label::String, overridden_label::Optional{String})
+
+    relation_type_set_relates_req = overridden_label !== nothing ?
+                                            Proto.RelationType_SetRelates_Req(;
+                                                label = role_label,
+                                                overridden_label)  :
+                                            Proto.RelationType_SetRelates_Req(;
+                                                label = role_label)
```

Comment:
Minor formatting issue. Only a single indent from prior line is needed.

Reply:
Hope, I changed it right yet.